### PR TITLE
Feature/Add Social Logo

### PR DESCRIPTION
**Pull request by AhlOct**
Preview link : https://raw.githack.com/AhlOct/open-source/feature/Add-Social-Logo/index.html

### CODE REVIEW

File: en_ID/navigation.md [hunk 1]
```diff
@@ -42,13 +42,23 @@
 
 [Social]()
 
+<<<<<<< HEAD
```
Comment (empeje) [inline en_ID/navigation.md]: I think this is coming from git conflict and not necessary.

**Comment by empeje** [comment]
Captura de Pantalla 2021-06-20 a la(s) 14 29 03

File: en_ID/navigation.md (hunk 1 above)
Comment (AhlOct) [reply en_ID/navigation.md]: oww, sorry. I fixed that.

**Comment by empeje** [comment]
Nicely done @AhlOct 🎉

**Pull request merged**